docker/common.inc.sh

```diff
@@ -312,3 +312,36 @@ deploy() {
 run() {
   exec_scripts "$WORKDIR/docker/entry.d/" "container bootstrap"
 }
+
+show_tests_help() {
+  echo "
+SYNOPSIS: Initializes the test environment in an easier way.
+
+USAGE: $0 $1 [<suite>[,<suite>,<suite>,...]]
+
+EXAMPLE:
+
+    - Running all suites:
+      $0 $1
+
+    - Running only one suite:
+      $0 $1 Acceptance
+
+    - Running more than one suites:
+      $0 $1 Acceptance,Unit,Functional
+"
+}
+
+tests() {
+  shift
+
+  case $1 in
+  '--help')
+    show_tests_help
+    ;;
+  *)
+    exec_scripts "$WORKDIR/docker/test.d/"
+    vendor/bin/codecept run $*
```

Comment:
How to execute tests should be configurable. May be an variable like `TEST_RUN_CMD="vendor/bin/codecept run"` in the `defaults.inc.sh` file.

Our aim with the base image is to provide a generic interface to common spryker tasks (run yves/zed, run tests, build the shop, ...) with meaningful defaults (in this case exec run for default codeception), but to also allow overriding the defaults.
So pipelines and other 3rd party tools and even developers have a common interface (like an API) to work with different shop implementations.